docs/components/store.md

```diff
@@ -291,6 +291,7 @@ type: IN-MEMORY
 config:
   max_size: 0
   max_item_size: 0
+enabled_items: []
```

Comment:
Let's add point about this in doc, with available options?

Reply:
Good point, I updated the doc.